src/components/MultiHopTrade/components/Approval/Approval.tsx

```diff
@@ -150,129 +146,134 @@ const ApprovalInner = ({
 
   return (
     <SlideTransition>
-      <Card variant='unstyled'>
-        <CardHeader textAlign='center' px={0} pt={0}>
-          <Heading>
-            <Text translation='assets.assetCards.assetActions.trade' />
-          </Heading>
-        </CardHeader>
-        <CardBody pb={0} px={0}>
-          <Flex
-            justifyContent='center'
-            alignItems='center'
-            flexDirection='column'
-            width='full'
-            as='form'
-            onSubmit={handleSubmit(approveContract)}
+      <CardHeader textAlign='center'>
+        <Heading as='h5'>
+          <Text translation='assets.assetCards.assetActions.trade' />
+        </Heading>
+      </CardHeader>
+      <CardBody>
+        <Flex
+          justifyContent='center'
+          alignItems='center'
+          flexDirection='column'
+          width='full'
+          as='form'
+          onSubmit={handleSubmit(approveContract)}
+        >
+          <CountdownCircleTimer
+            isPlaying={!!approvalTxId || isSubmitting}
+            size={90}
+            strokeWidth={6}
+            trailColor={theme.colors.whiteAlpha[500]}
+            duration={60}
+            colors={theme.colors.blue[500]}
+            onComplete={() => ({ shouldRepeat: true })}
           >
-            <CountdownCircleTimer
-              isPlaying={!!approvalTxId || isSubmitting}
-              size={90}
-              strokeWidth={6}
-              trailColor={theme.colors.whiteAlpha[500]}
-              duration={60}
-              colors={theme.colors.blue[500]}
-              onComplete={() => ({ shouldRepeat: true })}
+            {() => (
+              <Image
+                src={tradeQuoteStep.sellAsset.icon}
+                boxSize='60px'
+                fallback={<SkeletonCircle boxSize='60px' />}
+              />
+            )}
```

Comment:
this one can be put into a useCallback called something like `renderImage` and then passed in here